subsys/bluetooth/enocean.c

```diff
@@ -28,6 +28,9 @@ LOG_MODULE_REGISTER(bt_enocean);
 #define DATA_TYPE_LIGHT_LEVEL_SOLAR_CELL 0x04
 #define DATA_TYPE_OPTIONAL_DATA 0x3c
 
+#define DEVICE_TYPE_SWITCH 0xe215
```

Comment:
May be worth to move these macros to header files if someone would like to use them in the application level. Like hooking up the scanner callback and doing some UI prompting when commissioning packets are received.

Reply:
Sounds like out of scope, and they can easilly be moved later if we decide to expose this to the application. Please create a separate issue for this.

Reply:
NCSDK-27048